phd-tests/tests/src/cpuid.rs

```diff
@@ -48,21 +50,11 @@ async fn cpuid_instance_spec_round_trip_test(ctx: &Framework) {
 }
 
 #[phd_testcase]
-async fn cpuid_boot_test(ctx: &Framework) {
+async fn cpuid_boot_and_migrate_test(ctx: &Framework) {
     use cpuid_utils::bits::*;
 
-    // This test verifies that plumbing a reasonably sensible-looking set of
-    // CPUID values into the guest produces a bootable guest.
-    //
-    // The definition of "reasonably sensible" is derived from the suggested
-    // virtual-Milan CPUID definitions in RFD 314. Those are in turn derived
-    // from reading AMD's manuals and bhyve's source code to figure out what
-    // host CPU features to hide from guests.
-    //
-    // To try to make this test at least somewhat host-agnostic, read all of the
-    // host's CPUID leaves, then filter to just a handful of leaves that
-    // advertise features to the guest (and that Linux guests will check for
-    // during boot).
```

Comment:
this seems like information that we should hang onto here for future readers? or is it no longer true?

Reply:
It's obsolete as of #844, but I forgot to remove it in that PR. The test used to try to cobble together its own CPUID values that it thought would produce a bootable VM; now it just asks bhyve for the defaults that it would otherwise have supplied.